tsconfig.json

```diff
@@ -1,16 +1,11 @@
 {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "outDir": "out",
-        "lib": [
-            "es6"
-        ],
-        "sourceMap": true,
-        "rootDir": "."
-    },
-    "exclude": [
-        "node_modules",
-        ".vscode-test"
-    ]
-}
\ No newline at end of file
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "ES2020",
```

Comment:
This will cover VSCode from April 2021 (Electron 12 onwards), anything Node 14 or above.
You should be fine here I set your version in the package.json so anyone on an old version just won’t get the update.

Reply:
Cool, I was just curious. I'm not worried, I think vast majority of users are up-to-date with VSCode updates (I usually postpone updates but just by a week or two).